src/test/java/teammates/test/cases/ui/browsertests/AdminActivityLogPageUiTest.java

```diff
@@ -41,6 +50,18 @@ public void testContent() {
         Url logPageUrl = createUrl(Const.ActionURIs.ADMIN_ACTIVITY_LOG_PAGE);
         logPage = loginAdminToPage(browser, logPageUrl, AdminActivityLogPage.class);
         logPage.verifyIsCorrectPage();
+        
+        ______TS("content: navigate to other pages to get some logs");
+        logPage.navigateTo(createUrl(Const.ActionURIs.ADMIN_HOME_PAGE));
+        logPage.waitForPageToLoad();
+        logPage.navigateTo(createUrl(Const.ActionURIs.ADMIN_ACCOUNT_MANAGEMENT_PAGE));
+        logPage.waitForPageToLoad();
+        logPage.navigateTo(createUrl(Const.ActionURIs.ADMIN_SEARCH_PAGE));
+        logPage.waitForPageToLoad();
+        logPage.navigateTo(createUrl(Const.ActionURIs.ADMIN_ACTIVITY_LOG_PAGE));
+        logPage.waitForPageToLoad();
+        logPage.verifyHtmlMainContent("/adminActivityLogPage.html");
```

Comment:
Anyway, as you pointed out, this only happens if the number of logs is large and an ajax request is made. Maybe you can make a new issue for this and fix this before merging this in?